metricbeat/module/logstash/node_stats/node_stats.go

```diff
@@ -102,15 +97,27 @@ func (m *MetricSet) Fetch(r mb.ReporterV2) error {
 	return nil
 }
 
-func (m *MetricSet) init() error {
-	if m.XPack {
-		err := m.CheckPipelineGraphAPIsAvailable()
-		if err != nil {
-			return err
-		}
+func (m *MetricSet) setServiceURI() error {
```

Comment:
Actually I would cover this method with a unit test to shorten debugging time (missing query parameters, etc.).

I'm not sure if the method name is not misleading (looks like a setter), but it enables Graph API, right?